rebuild-manager.js

```diff
@@ -1,3 +1,5 @@
+// TODO: remove this file?
```

Comment:
@mixmix is this ok? since db2 is like 10x faster i felt like the complexity that this manager adds is maybe not worth it atm

Reply:
YES